backend/service/authn/oidc.go

```diff
@@ -175,6 +184,15 @@ func (p *OIDCProvider) Verify(ctx context.Context, rawToken string) (*Claims, er
 		return nil, err
 	}
 
+	// If the token doesn't exist in the token storage anymore, it must have been revoked.
+	// Fail verification in this case.
+	if p.tokenStorage != nil {
+		token, err := p.tokenStorage.Read(ctx, claims.Subject, clutchProviderName)
+		if token == nil {
+			return nil, err
+		}
+	}
```

Comment:
do you mind deploying this in a fast-follow to avoid everyone getting logged out at the same time? it will also allow us to verify the tokens are being stored correctly in the db upon exchange.

Reply:
Sure, removed this piece of code and updated the test for now, will follow up with another PR.